tigershark/facade/__init__.py

```diff
@@ -295,6 +297,25 @@ def loops(self, theClass, anX12Message, *args, **kwargs):
         return [theClass(loop, *args, **kwargs) for loop in
                 anX12Message.descendant("loop", theClass.loopName)]
 
+    def to_dict(self):
+        return Facade._to_python_dict(self)
+
+    def to_json(self, **kwargs):
+        return json.dumps(self.to_dict(), **kwargs)
+
+    @staticmethod
+    def _to_python_dict(instance):
+        properties = [x for x in inspect.getmembers(instance) if not x[0].startswith("_")]
+        record = dict()
+        for k, v in properties:
+            if isinstance(v, list):
```

Comment:
What if none of these clauses match? I think it would be good to handle that in some way, or throw an error.